.storybook/stories/SubmitButton.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import type { ComponentMeta, ComponentStory } from "@storybook/react";
+
```

Comment:
I think it might be the organizeImports not running automatically on save instead of prettier. I'll take a look for sure.